rainier-tests/src/test/scala/com/stripe/rainier/cats/suites.scala

```diff
@@ -1,13 +1,45 @@
-package com.stripe.rainier.cats
+package com.stripe.rainier
```

Comment:
I did this so that I could get access to the `rainier` scope that the various `Real` instances were defined in.